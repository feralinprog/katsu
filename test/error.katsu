# cause a signal to be raised
1 / 0
