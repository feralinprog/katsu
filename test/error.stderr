terminating program: could not load module
